internal/compiler-bridge/src-2.10/main/scala/xsbt/API.scala

```diff
@@ -1,81 +0,0 @@
-/* sbt -- Simple Build Tool
```

Comment:
We should cross publish util-logging and util-testing for 2.10 if that's what it takes.